src/test/java/no/unit/nva/cristin/institutions/utils/AdapterTest.java

```diff
@@ -0,0 +1,135 @@
+package no.unit.nva.cristin.institutions.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import no.unit.nva.cristin.institutions.model.Identifier;
+import no.unit.nva.cristin.institutions.model.UnitObject;
+import no.unit.nva.cristin.institutions.model.UnitObjectBuilder;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Type;
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class AdapterTest {
+
+    @Disabled
```

Comment:
Some comment, why they are disabled and what do we need to do to enable them. Or delete them?